package-lock.json

```diff
@@ -31,15 +31,16 @@
         "@typescript-eslint/eslint-plugin": "^4.28.3",
         "@typescript-eslint/parser": "^4.28.3",
         "chai": "^4.3.4",
-        "eslint": "^7.30.0",
+        "eslint": "^7.32.0",
         "eslint-config-prettier": "^8.3.0",
         "mocha": "^8.4.0",
         "rimraf": "^3.0.2",
         "ts-node": "^9.1.1",
         "typescript": "^4.3.5"
       },
       "engines": {
-        "node": "*"
+        "node": ">=12.0",
```

Comment:
Why wasn't this added in previous lockfile updates?

Reply:
I wondered the same when I seen it before uploading.

My guess is that yesterday we merged https://github.com/ansible/ansible-language-server/commit/1514f0fee1ba454c0cdc70473ae419dfe86a4fa5 we did not had a pipeline setup.

Reply:
I don't think so. Your last merge today doesn't fail: https://github.com/ansible/ansible-language-server/runs/3632442387. It would've failed if this was the case.

Reply:
I think the most plausible explanation is that `npm ci` probably only complains about the deps and not other metadata :man_shrugging: